x-pack/plugins/security_solution/scripts/endpoint/common/roles_users/serverless/es_serverless_resources/README.md

```diff
@@ -0,0 +1,48 @@
+# Security Solution Serverless Resources
+
+Directory contains ES serverless resources that can be used to override the defaults that are loaded when ES is started in serverless mode. For more information on how these are used [packages/kbn-es/src/serverless_resources/README.md](https://github.com/elastic/kibana/blob/main/packages/kbn-es/src/serverless_resources/README.md)
+
+> **ℹ️ NOTE**
+> 
+> The files referenced via `--resources` argument will be bound and mounted to the ES docker containers that are running ES. This means that any changes to the files done on the host machine will be automatically (after a delay - 5s by default) picked up by Elasticsearch and applied to the ES docker nodes.
+
+## Usage
+
+Example executed from the root directory of Kibana: 
+
+```shell
+yarn es serverless \
+--clean \
+--teardown \
```

Comment:
`--teardown` is not in use anymore and enabled by default.

Reply:
Thanks. I will remove 👍